### PR TITLE
Foundation styling

### PR DESCRIPTION
**Pull request by sublimemarch**

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -1,45 +1,55 @@
...
+      </div>
+    </div>
   </div>
-  <!-- Needs a new source! -->
-  <!-- <img src="" class="one-third-image"> -->
-</div>
 
+  <div class="faceook-box">
+    <h3>Follow us on Facebook!</h3>
+    <div class="fb-page" data-href="https://www.facebook.com/pages/Humble-Hearts-Organization/477172595731068?" data-width="500" data-height="300" data-hide-cover="false" data-show-facepile="true" data-show-posts="true"><div class="fb-xfbml-parse-ignore"><blockquote cite="https://www.facebook.com/pages/Humble-Hearts-Organization/477172595731068"><a href="https://www.facebook.com/pages/Humble-Hearts-Organization/477172595731068">Humble Hearts Organization</a></blockquote></div></div>
```
Comment (sublimemarch) [inline app/views/welcome/index.html.erb:47]: This facebook box isn't showing and I can't figure out why. I'm using the code as facebook tells me to use it. I remember it was working during the hackathon.

Comment (SStrausman) [inline app/views/welcome/index.html.erb:47]: Yeah for some reason your box was the only one it wasnt working on at the
hackathon. I don't know how you could test if its your machine or a general
issue.

Comment (SStrausman) [inline app/views/welcome/index.html.erb:47]: Was the facebook box working for you on local host before you started working on this particular improvement?

Comment (sublimemarch) [inline app/views/welcome/index.html.erb:47]: No, it wasn't working on localhost. I didn't change much of anything on the facebook box code except for its size.

Comment (SStrausman) [inline app/views/welcome/index.html.erb:47]: I think we can just assume a continued issue with your machine then. It does make it hard for you to style it though...

Comment (sublimemarch) [inline app/views/welcome/index.html.erb:47]: Lucky me, there's no styling that needs to be done on the facebook box. :D

**Comment by SStrausman** [comment]
:+1:

**Pull request merged**